manifests/device-plugin-mps.yml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
```

Comment:
Can you add some comment why this manifest is needed?

Reply:
Oh.. I think we want to have two manifest

- device-plugin-mps.yml
- device-plugin.yml  - by default disable MPS

em. I think we probably want to enable MPS by default and add a non-mps one? What's your opinion? @walkley

Reply:
I would suggest you submit few separate PRs.

1. change to apps/v1 to support 1.16 and above. 
2. Add  batch test job
3. Add non-mps template.